### PR TITLE
Update output way if there is playlist

### PR DESCRIPTION
**Pull request by arminsalcin**
So I used this very useful tool but when I tried to download the playlist all the files were right in my download folder which sometimes can be a bit annoying to mix with all my files.

I made this PR so if there is a playlist we make a folder with the name of the playlist and store all music files there.

If for some reason we can't get the playlist name the default fallback will be "playlistOutput" If you think there is a better name for this one it can be changed.

Btw I didn't run any builds locally so it should be tested after the build too.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "axios": "^1.6.0",
     "browser-id3-writer": "^4.4.0",
     "electron-updater": "^5.2.1",
-    "ffmpeg-static": "4.2.7",
+    "ffmpeg-static": "^5.2.0",
     "fluent-ffmpeg": "^2.1.2",
```
Comment (arminsalcin) [inline package.json:33]: I needed to update ffmpeg-static to the latest to be able to do npm i on the "M1" processor, this should be evaluated by build if there are some breaking changes, I checked them but there is none described on the package itself.

**Comment by fajicbenjamin** [comment]
Thanks for the PR, first of all!
I just tried it out and like it very much! I tried it locally on my Windows machine and Mac (Intel) and works fine on both. 

Although new ffmpeg binaries seem to convert file bit slower on Windows (no benchmark, just my subjective feeling). On Mac, even before it was bit slower than Windows for me, so seems just the same. I do experience weird -random_big_number% when starting conversion (after 50% progress) before continuing normally on both systems.

Most likely would also need to update Github action as I believe new ffmpeg-static version would require node 16. I am wondering, do current releases I publish work normally on M1 Mac? If yes, then I would like to keep old binaries version for some more time. Please let me know (sorry for bit worse development experience for you in that case, as you would locally need to install newer binaries but not push them 😞)

**Comment by arminsalcin** [comment]
> Thanks for the PR, first of all! I just tried it out and like it very much! I tried it locally on my Windows machine and Mac (Intel) and works fine on both.
> 
> Although new ffmpeg binaries seem to convert file bit slower on Windows (no benchmark, just my subjective feeling). On Mac, even before it was bit slower than Windows for me, so seems just the same. I do experience weird -random_big_number% when starting conversion (after 50% progress) before continuing normally on both systems.
> 
> Most likely would also need to update Github action as I believe new ffmpeg-static version would require node 16. I am wondering, do current releases I publish work normally on M1 Mac? If yes, then I would like to keep old binaries version for some more time. Please let me know (sorry for bit worse development experience for you in that case, as you would locally need to install newer binaries but not push them 😞)

Happy you liked it, i imagine that currently updating the package is not the best idea, not a big deal then.

I reverted the commit.

Also, your release version perfectly works with M1 so no need to update the package yet.

**Comment by fajicbenjamin** [comment]
Thank you. Glad to merge it. Will be available in 0.4.8 version soon

**Pull request merged**